DQMOffline/Muon/python/muonAnalyzer_cff.py

```diff
@@ -42,3 +42,16 @@
                                   MediumMuonEfficiencyAnalyzer*
                                   TightMuonEfficiencyAnalyzer*                                
                                   muonPFsequence)
+
+from Configuration.StandardSequences.Eras import eras
+for e in [eras.pA_2016]:
+  e.toModify(diMuonHistos, etaBin = cms.int32(175))
+  e.toModify(diMuonHistos, etaBBin = cms.int32(175))
+  e.toModify(diMuonHistos, etaEBin = cms.int32(175))
```

Comment:
Also, as was pointed earlier, this is not the right place to edit dimuonHistos. Era modifications have to be done in the file that defines corresponding module or PSet.
Move these edits to 
DQMOffline/Muon/python/diMuonHistograms_cfi.py